lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/handler/DrivingPermitHandler.java

```diff
@@ -64,6 +65,8 @@
     private final ConfigurationService configurationService;
     private final AuditService auditService;
 
+    private static final int MAX_ATTEMPTS = 2;
```

Comment:
Have we got a TODO to make this a parameter?

Reply:
Re-added the todo comment